### ISSUE TITLE
SeaDroneSim: Simulation of Aerial Images for Detection of Objects Above Water. (arXiv:2210.16107v1 [cs.CV])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#23375]
SeaDroneSim: Simulation of Aerial Images for Detection of Objects Above Water. (arXiv:2210.16107v1 [cs.CV])<br>
https://ift.tt/7LgJaiO<br>
<p>Unmanned Aerial Vehicles (UAVs) are known for their fast and versatile applicability. With UAVs' growth in availability and applications, they are now of vital importance in serving as technological support in search-and-rescue(SAR) operations in marine environments. High-resolution cameras and GPUs can be equipped on the UAVs to provide effective and efficient aid to emergency rescue operations. With modern computer vision algorithms, we can detect objects for aiming such rescue missions. However, these modern computer vision algorithms are dependent on numerous amounts of training data from UAVs, which is time-consuming and labor-intensive for maritime environments. To this end, we present a new benchmark suite, \textit{\textbf{SeaDroneSim}}, that can be used to create photo-realistic aerial image datasets with the ground truth for segmentation masks of any given object. Utilizing only the synthetic data generated from \textit{\textbf{SeaDroneSim}}, we obtain 71 mAP on real aerial images for detecting BlueROV as a feasibility study. This result from the new simulation suit also serves as a baseline for the detection of BlueROV.</p>
<br>
<br>
via cs.RO updates on arXiv.org http://arxiv.org/<br>
